TypeCobol.Test/TestCollection.cs

```diff
@@ -160,8 +160,8 @@ private void CheckParserCobol85(bool cobol)
             foreach (string directory in Directory.GetDirectories(root))
             {
                 var dirname = Path.GetFileName(directory);
-
-			    Console.WriteLine("Entering directory \"" + dirname + "\" [" + string.Join(", ", extensions) + "]:");
+                int number = Path.GetFileNameWithoutExtension(directory).Count();
```

Comment:
Test code to be removed ?

Reply:
Yes. @laabidiHend could you please revert this file to its original state ?